### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by SixtyFiveOhTwo**
fixed errors from massive merge on 2/5 robot functions as it did before now

#6

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -20,36 +20,26 @@
...
+    // Joystick axis 
     public static final int LEFT_AXIS = 1;
     public static final int RIGHT_AXIS = 5;
     public static final double DEAD_ZONE = 0.3;
 
-    public static final int JOYSTICK_X_AXIS = 0;
-    public static final int JOYSTICK_Y_AXIS = 1;
-    public static final int JOYSTICK_Z_AXIS = 2;
     // color sensor stuff
     public static final double colorSensorId0 = 0;
```
Comment (Gliu1211) [inline src/main/java/frc/robot/Constants.java]: Remove unused colorSensor Ids

**Review by Gliu1211 — CHANGES_REQUESTED** [review]
Review changes.

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -20,36 +20,26 @@
...
-    // Joystick Control Stuff
+    // Joystick axis 
     public static final int LEFT_AXIS = 1;
     public static final int RIGHT_AXIS = 5;
     public static final double DEAD_ZONE = 0.3;
 
-    public static final int JOYSTICK_X_AXIS = 0;
-    public static final int JOYSTICK_Y_AXIS = 1;
-    public static final int JOYSTICK_Z_AXIS = 2;
     // color sensor stuff
```
Comment (Gliu1211) [inline src/main/java/frc/robot/Constants.java]: Revise comment to be more specific

**Review by Gliu1211 — CHANGES_REQUESTED** [review]
review change

**Review by Gliu1211 — APPROVED** [review]

**Review by Gliu1211 — APPROVED** [review]

**Review by Gliu1211 — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/ColorSensorSubsystem.java [hunk 3]
```diff
@@ -1,104 +0,0 @@
...
-import edu.wpi.first.wpilibj.I2C;
-
-import edu.wpi.first.wpilibj.util.Color;
-
-import com.revrobotics.ColorSensorV3;
-import com.revrobotics.ColorMatchResult;
-import com.revrobotics.ColorMatch;
-
-
-public class ColorSensorSubsystem extends SubsystemBase {
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/subsystems/ColorSensorSubsystem.java:20]: Restore this file.

File: src/main/java/frc/robot/commands/ColorSensorCommand.java [hunk 4]
```diff
@@ -1,55 +0,0 @@
...
-package frc.robot.commands;
-
-import edu.wpi.first.wpilibj2.command.CommandBase;
-
-import frc.robot.subsystems.ColorSensorSubsystem;
-
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-
-
-public class ColorSensorCommand extends CommandBase {
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/commands/ColorSensorCommand.java]: Restore this file.

**Review by liam-a-wms — CHANGES_REQUESTED** [review]

**Review by liam-a-wms — APPROVED** [review]

**Review by Gliu1211 — COMMENTED** [review]
nice code style

**Pull request merged**